### DISCUSSION TITLE
MuseScore - JACK - Ableton Live

### DISCUSSION THREAD

**Discussion by SoutherlyBuster** [jackaudio/jack2#751] [Q&A]
What I want to do, is to route the midi output from Musescore to Ableton Live. I assume this can be done?

I installed Jack 1.9.18 on my Mac's OSX using the 64 installer. I can enable Jack in Musescore (scoring tool) and I hear the sound as it should do. I use the QjackCtl to set up Jack and to see the connections available (in the Graph window), which creates the "mscore" which is the output of Musescore and if routed to "system - playback_1" and "system - playback_2", sound are heard. In Ableton Live in the Preferences - Audio, the DriverType is left as CoreAudio (drop down list only gives NoAudio or CoreAdudio), the Audio Input and Output lists the audio inputs and outputs; the Preferences - LinkMidi, where I set where my midi is coming from, can not see anything that looks like Jack. 

So it appears the Ableton Live is unaware of Jack, neither Jack audio nor Jack midi.

I read some older posts of people trying to get Jack and Ableton Live to work together and they said only the 32 bit version of Jack will do it. But now only 64bit and Universal installer available.

I am using Live version 10.

Some help here would be appreciated. Thanks in advance.

**Comment by falkTX** [comment]
Old methods (that should be noted only worked on older versions of macOS) used something known as jack-router.
jack-router for macOS no longer works since Catalina, and no one is really working on jack-router anymore anyway.

While JACK builds for Windows added back jack-router in 1.9.17, this was without any modifications done to the old code.
It is just because it still mostly works on Windows, so there is some usefulness to it.

As jack-router does not work on macOS anymore, so you have to find something else for it.

The easiest path here, since musecore is mainly about writing a music score / composition and not full mixing (not yet!), is to use a virtual MIDI tool.
JACK does not provide this, you have to find something else.

**Reply by SoutherlyBuster** [reply]
Thanks for the reply falkTX and suggestions on the way forward.
